library/src/main/java/com/novoda/downloadmanager/lib/Downloads.java

```diff
@@ -26,8 +26,50 @@
  *
  * @pending
  */
-final class Downloads {
-    private Downloads() {
+public class Downloads {
+
+    private final String authority;
+
+    /**
+     * The content URI for accessing publicly accessible downloads (i.e., it requires no
+     * permissions to access this downloaded file)
+     */
+    public Uri getPublicityAccessibleDownloadsUri() {
```

Comment:
I don't think this is a future PR. We've made this class public here (which it wasn't before), and given this makes this part of the public API, we should move this functionality to a better-named class, IMO.

also `getPublicityAccessibleDownloadsUri` -> `getPubliclyAccessibleDownloadsUri ` and user knows its publicly accessible because they can access it :tongue: so probably don't need it in the name